src/RuleSet/RuleSet.php

```diff
@@ -65,11 +65,15 @@ public static function parseRuleSet(ParserState $parserState, RuleSet $ruleSet):
         while ($parserState->comes(';')) {
             $parserState->consume(';');
         }
-        while (!$parserState->comes('}')) {
+        for (;;) {
```

Comment:
Let's communicate the intention behind this line more clearly:

```suggestion
        while (true) {
```